File: pgm/ABC_Kotlin/ABC_422_A.kt
// import kotlin.math.*

const val one: Int = 1
const val two: Int = 2
const val eight: Int = 8
const val zero: Int = 0

fun main() {
    val S: String = readln()
    var world: Int = S[zero].digitToInt()
    var stage: Int = S[two].digitToInt()
    if (stage == eight) {
        world++
        stage = one
    } else {
        stage++
    }
    println("$world${S[one]}$stage")
    kotlin.system.exitProcess(0)
}

File: pgm/ABC_Dart/ABC_422_A.dart
// import 'dart:collection';
import 'dart:io';
// import 'dart:math';

void main() {
  final String S = stdin.readLineSync()!;
  const int one = 1;
  const int two = 2;
  const int eight = 8;
  const int zero = 0;
  int world = int.parse(S[zero]);
  int stage = int.parse(S[two]);
  if (stage == eight) {
    world++;
    stage = one;
  } else {
    stage++;
  }
  print("$world${S[one]}$stage");
  exit(0);
}
